.travis.yml

```diff
@@ -0,0 +1,35 @@
+dist: focal
+
+arch:
+  - amd64
+
+os: linux
+
+language: shell
+
+notifications:
+  email:
+  - team@appwrite.io
+
+before_script: docker run --rm --interactive --tty --volume "$(pwd)":/app composer install --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+
+before_install:
+- curl -fsSL https://get.docker.com | sh
+- echo '{"experimental":"enabled"}' | sudo tee /etc/docker/daemon.json
+- mkdir -p $HOME/.docker
+- echo '{"experimental":"enabled"}' | sudo tee $HOME/.docker/config.json
+- sudo service docker start
+- >
+  if [ ! -z "${DOCKERHUB_PULL_USERNAME:-}" ]; then
+    echo "${DOCKERHUB_PULL_PASSWORD}" | docker login --username "${DOCKERHUB_PULL_USERNAME}" --password-stdin
+  fi
+- docker --version
+
+install:
+- docker-compose up -d
+- sleep 10
+
+script:
+- docker ps
+- docker-compose exec tests vendor/bin/phpunit --configuration phpunit.xml
+- docker-compose exec tests vendor/bin/psalm --show-info=true
```

Comment:
In the .gitpod.yml file we use the newer `docker compose` instead of `docker-compose` 

Since `docker compose` is the way to go going forward, can we use the newer syntax for all future work and libraries ?